### ISSUE TITLE
Replace mentions to l1 and l2 to 'ethereum' and 'starknet'

### ISSUE THREAD

**Issue by achab** [aave-starknet-project/aave-starknet-bridge#43]

**Comment by achab** [comment]
@JorikSchellekens Not sure to understand the scope of this issue.

Do you mean in the code? Like rename `tokenProxyL1` to `tokenProxyEthereum`?

**Comment by JorikSchellekens** [comment]
@achab I was considering whether the ecosystem has acclimatised to l1 / l2 nomenclature or if it'd be clearer to use ethereum / starknet. I'm not sure which is the better option but we should be consistent across the board. For example ETHstaticAToken might better be named l1StaticAToken. (Or actually in that case maybe just staticAToken but that's beside the point).

**Comment by achab** [comment]
@JorikSchellekens good point, I know that maker-dai-bridge is using l1/l2 nomenclature. IMO we should only mention the chain where there is an ambiguity i.e. for contracts that have their counterpart on the other chain, or for the tests where contracts of the two chains are used. Using l1/l2 might be simpler and clear enough for variables like `l1_token` or `l2_recipient`.

**Comment by achab** [comment]
@JorikSchellekens so I'd rather propose we replace ethereum with l1 and starknet with l2

**Comment by JorikSchellekens** [comment]
Sounds good, let's do that
